web/src/app/tenants/hooks.ts

```diff
@@ -1,44 +1,71 @@
 import { FormikHelpers } from "formik";
-import { SearchRoom } from "../rooms/hooks";
+import { RoomsFormValues, SearchRoom } from "../rooms/hooks";
 import { ActionButtonProps, Column, TableActions } from "@/components/Table";
-import { useCallback, useState } from "react";
-import { SelectFieldProps } from "@/components/Select";
+import { useCallback, useEffect, useState } from "react";
+import { OptionSelect, SelectFieldProps } from "@/components/Select";
 import { InputFieldProps, Field } from "@/components/hooks/useModal";
 
-interface TenantFormValues {
+interface RoomHistoryValues {
+  id: number;
+  roomId: number;
+  userId: number;
+}
+
+/**
+ * TenantFormValues properties
+ * extends of {@link RoomHistoryValues}  pick only property roomId
+ * extends of {@link RoomsFormValues} pick only property amount
+ */
+interface TenantFormValues
+  extends Partial<Pick<RoomHistoryValues, "roomId">>,
```

Comment:
optional naman daan ang `roomId`
```suggestion
  extends Pick<RoomHistoryValues, "roomId">,
```